packages/server/src/auth/auth.controller.ts

```diff
@@ -417,6 +417,8 @@ export class AuthController {
         sid ? sid : -1,
         organizationId,
       );
+      // create student subscriptions
+      await this.authService.createStudentSubscriptions(userId);
```

Comment:
note that register() is not the only method where users are created. Login with sso as well as login with google will create a user account if they don't have one yet.